master/cmd/determined-master/root.go

```diff
@@ -79,13 +79,26 @@ func initializeConfig() error {
 	if err != nil {
 		return err
 	}
-	if err = mergeConfigBytesIntoViper(bs); err != nil {
-		return err
+
+	// Write a configMap from the config file, and create a copy (cpMap) to
+	// deepcopyq values needed to override viper's merge auto-lowercasing.
+	var configMap, cpMap map[string]interface{}
+	configErr := yaml.Unmarshal(bs, &configMap)
+	cpErr := yaml.Unmarshal(bs, &cpMap)
+	if configErr != nil || cpErr != nil {
+		return errors.Wrap(err, "error unmarshal yaml configuration file")
```

Comment:
```err``` here will always be nil, since we are setting ```configErr``` / ```cpErr``` not ```err```

I think checking the errors one at at a time while more verbose is easier to scan over quickly.

Reply:
oh oops, that `err` is leftover from previous implementation.